properties/src/main/java/gov/cms/ab2d/properties/service/PropertiesAPIServiceImpl.java

```diff
@@ -1,34 +1,84 @@
 package gov.cms.ab2d.properties.service;
 
+import gov.cms.ab2d.properties.client.PropertiesClient;
+import gov.cms.ab2d.properties.client.Property;
 import gov.cms.ab2d.properties.dto.PropertiesDTO;
-import gov.cms.ab2d.properties.model.Properties;
-import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+import gov.cms.ab2d.properties.client.PropertiesClientImpl;
 
 import java.util.List;
 
 @Slf4j
 @Service
-@AllArgsConstructor
 public class PropertiesAPIServiceImpl implements PropertiesAPIService {
     private PropertiesService propertiesService;
+    private boolean usePropertyService;
+    private PropertiesClient propertiesClient;
+    private static final String ERROR_MESSAGE = "Cannot access properties service, using default database value";
+
+    PropertiesAPIServiceImpl(@Value("${feature.property.service.enabled:false}") boolean usePropertyService, PropertiesService service,
+                             @Value("${property.service.url}") String propertyServiceUrl) {
+        this.propertiesService = service;
+        this.usePropertyService = usePropertyService;
+        this.propertiesClient = new PropertiesClientImpl(propertyServiceUrl);
+    }
 
     @Override
     public String getProperty(String property) {
-        Properties properties = propertiesService.getPropertiesByKey(property);
-        return properties.getValue();
+        String value;
+        if (usePropertyService) {
+            try {
+                Property prop = propertiesClient.getProperty(property);
+                value = prop.getValue();
```

Comment:
can't these two be one line. Don't need 2 variables. Or at least a check if properties is null